values.yaml

```
@@ -440,6 +441,11 @@ rabbitmq-ha:
   rabbitmqUsername: admin
   # TODO: Use default random 24 character password, but need to fetch this string for use by downstream services
   rabbitmqPassword: 9jS+w1u07NbHtZke1m+jW4Cj
+  # RabbitMQ available memory; default values might not be enough for StackStorm deployment to work properly (see: https://github.com/helm/charts/issues/12730)
+  rabbitmqMemoryHighWatermarkType: absolute
+  rabbitmqMemoryHighWatermark: 512MB
```

Please keep the values commented as at this moment we're just recommending these settings to our users.

```suggestion
  # RabbitMQ Memory high watermark. See: http://www.rabbitmq.com/memory.html
  # Default values might not be enough for StackStorm deployment to work properly. We recommend to adjust these settings for you needs as well as enable Pod memory limits via "resources".
  #rabbitmqMemoryHighWatermark: 512MB
  #rabbitmqMemoryHighWatermarkType: absolute
```

---

Agree!